app/src/main/java/org/luxoft/sdl_core/BluetoothLongWriter.java

```diff
@@ -59,12 +64,15 @@ public void onLongMessageSent() {
         triggerNextMessageProcessing();
     }
 
-    public void processWriteOperation(final byte[] message) {
-        Log.d(TAG, "Going to send message of size " + message.length);
-
+    public void processWriteOperation(final byte[] message_) {
```

Comment:
@VadymLuchko pls don't use underscore as it looks confusing like a variable of class
```suggestion
    public void processWriteOperation(final byte[] outgoing_message) {
```

Reply:
done